src/Features/CSharp/Portable/QuickInfo/CSharpSemanticQuickInfoProvider.cs

```diff
@@ -88,6 +88,17 @@ protected override ImmutableArray<TaggedText> TryGetNullabilityAnalysis(Workspac
 
             var typeInfo = semanticModel.GetTypeInfo(bindableParent, cancellationToken);
 
+            if (typeInfo.Type.IsValueType)
```

Comment:
how do you know .Type is not null?

Reply:
It's not called `Type?` 😋 

It could be, I supposed. I assumed with the kind check above it might be alright, but that's an educated guess and not a fact.

Reply:
hrmm............   you raise a good point.  It may be the case that iif you have any of those symbols and its any of the kinds checked that you will always have a type...

However, i would not risk it.  GetTypeInfo absolutely returns null, and i don't think there's a guaranteed invariant between the two that you can trust.

And yes, it should be `Type?` :)

Reply:
And if you're wondering @CyrusNajmabadi yes we're chatting about beginning to annotate the compiler APIs as well...